python/test/unit/language/test_standard.py

```diff
@@ -3,7 +3,7 @@
 import torch
 import triton.language as tl
 
-from test_core import _test_binary, int_dtypes, uint_dtypes, float_dtypes, numpy_random
+from .test_core import _test_binary, int_dtypes, uint_dtypes, float_dtypes, numpy_random
```

Comment:
Oh, I think we don't have `__init__.py`, so this is actually correct

Reply:
Running `pytest language/test_standard.py` from `python/test/unit` (or in other directories, amending the path specified in the `pytest` command) produces:
```
ImportError while importing test module '/home/danzimm/tross/triton/python/test/unit/language/test_standard.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
language/test_standard.py:6: in <module>
    from .test_core import _test_binary, int_dtypes, uint_dtypes, float_dtypes, numpy_random
E   ImportError: attempted relative import with no known parent package
```

I might have a misunderstanding, but I think implicit relative imports like this went away with python3